vrc-get-gui/locales/en.json5

```
@@ -6,6 +6,9 @@
     "vpm repositories": "VPM Repositories",
     "logs": "Logs",
     "sidebar:toast:version copied": "Version name was copied.",
+    "sidebar:bad hostname": "Bad hostname",
+    "sidebar:dialog:bad hostname": "The hostname is not good.",
```

```suggestion
    "sidebar:dialog:bad hostname": "Your hostname is not good.",
```

---

Odd wording, you should use "Entered hostname is invalid".

---

the hostname is already configured and not entered in ALCOM so "Entered" is odd I think. 
In addition, hostname with non ASCII is not invalid but may not be incompatible with unity so not good. (bad?)

"Your hostname (PC name) is not good" or "Your PC hostname is bad"?

---

Oh, I wasn't 100% clear about the context of this key.
Maybe "There are issues with your hostname." or something like that.
Something that isn't "is not good", and "bad" sounds judgmental.

---

Thank you for suggestion!

```suggestion
    "sidebar:dialog:bad hostname": "There are issues with your hostname.",
```